### PR TITLE
Remove historical routing.

### PR DESCRIPTION
**Pull request by ashercodeorg**
Google Analytics and Google Search (via the `link` parameter [per below]) were used to confirm these URLs are not seeing traffic in the past year.

### CODE REVIEW

**Comment by ashercodeorg** [comment]
CC @alicesteinglass @ryansloan @poorvasingal @joshlory @breville

**Comment by ashercodeorg** [comment]
CC @tanyaparker

**Comment by breville** [comment]
Seems reasonable to me, but are we sure that GA would catch all of these?  (i.e. none of them go to redirects that render a different URL and therefore GA wouldn't catch it?)

Also worth searching on google with the link: selector to see if we can find prominent sites still linking to these URLs.

In theory Google Webmaster Tools also has the ability to list which sites link to ours, but not sure how granular, because we have blank results.  I've sent you an invite to it, at any rate.

**Comment by ashercodeorg** [comment]
Looking at https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=link:studio.code.org%2Fk8intro+-site:code.org+-site:studio.code.org and similar, I'm not seeing anything that would give me cause to pause. Great idea, though, breville.

**Comment by ashercodeorg** [comment]
@breville Note that I did not receive an invite to Google Webmaster Tools, though I appear to have access (not sure if this was the result of any actions you took). Though GWT doesn't seem helpful for this, some of its functionality does seem like something we might want to investigate using.

File: dashboard/config/routes.rb [hunk 1]
```diff
@@ -229,11 +218,7 @@ def redirect_to_teacher_dashboard
   post '/admin/gatekeeper/set', :to => 'dynamic_config#gatekeeper_set', as: 'gatekeeper_set'
   get '/admin/:action', controller: 'reports', as: 'reports'
 
-  get '/stats/usage/:user_id', to: redirect_to_teacher_dashboard
-  get '/stats/students', to: redirect_to_teacher_dashboard
-  get '/stats/:user_id', to: redirect_to_teacher_dashboard
   get '/popup/stats', to: 'reports#header_stats', as: 'header_stats'
-  get '/redeemprizes', to: 'reports#prizes', as: 'my_prizes'
```
Comment (joshlory) [inline dashboard/config/routes.rb]: Actually I don't think we can remove this yet.  There are still references to this route [in the user menu](https://github.com/code-dot-org/code-dot-org/blob/479ee2fb1803ae79e349ff0ccf2fcfbcfcfc0c40/dashboard/app/views/shared/_user_header.html.haml#L13-L14).

These prizes predate the 2014 Hour of Code.  Maybe we shouldn't remove them from the DB, but should we remove the UI for accessing them?  The prizes for the past two years have been issued from Pegasus.

**Comment by jeremydstone** [comment]
Modulo Josh's comments above, and as long as you're sure all affected routes have received zero (not low) traffic for a full year, LGTM

File: dashboard/config/routes.rb (hunk 1 above)
Comment (ashercodeorg) [inline dashboard/config/routes.rb]: Done.

**Pull request merged**